tests/cases/config.yaml

```diff
@@ -93,6 +93,8 @@ Environments:
       facility_devices_user1: add_your_info
       facility_devices_user1_pwd: add_your_info
       facility_devices_user1_number: add_your_info
+      facility_devices_user1_name: add_your_info
```

Comment:
Is  this going to be needed for user2 as well?

Reply:
No, after talking to Michelle, I should change that to name of requester